spec/models/user_spec.rb

```diff
@@ -53,10 +56,24 @@
     expect(user.contacts).to include(contact)
   end
 
-  describe 'contact requests' do
-    let(:contact) { FactoryBot.create :user }
-    let(:request) { FactoryBot.create :user }
+  it 'has no relationship to social accounts' do
+    user_with_no_social_accounts = FactoryBot.build(:user)
+    expect(user_with_no_social_accounts.social_accounts).to be_empty
+  end
+
+  # rubocop:disable RSpec/MultipleExpectations
```

Comment:
Same as above - I don't really see the sense of overriding rubocop rules locally instead of adjusting them globally. :)

Reply:
I would just omit `expect(user_with_social_accounts.social_accounts).not_to be_empty`.  
We allowed two expectations in `.rubocop.yml` and this one is redundant because the array is not empty if the first element matches `GitHub`.